src/App.css

```diff
@@ -33,5 +36,103 @@ body {
   padding: 2px;
 }
 h1 {
-  font-size: 20px;
+  font-size: 20px; 
+  font-weight: 600;
+  border-bottom: 2px solid #090707;
+}
+.friends {
+  width: auto;  
+  height: 100px;
+  color: #070300;
+}
+
+.list-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0; 
+  border-top: 2px solid #27221e;
+  padding: 5px;
+  margin: 0px -10px 0px -10px;
```

Comment:
Negative margin discouraged. Do you really need this ? Please check if there is any other way to achieve the layout